foxpuppet/foxpuppet.py

```diff
@@ -3,11 +3,11 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-from .windows import WindowManager
+from .windows.window_manager import WindowManager
```

Comment:
I think it would be nice for us to be able to import `WindowManager` directory from the `windows` module. We should be able to do this by adding the import to `windows/__init__.py`. I think we could do this with `BaseWindow` and maybe even `BrowserWindow` too. This is really just cosmetic, though. What do you think @whimboo?